docs/RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Go-IPFS Changelog Template
```

Comment:
Broke this out as it doesn't really belong in the release _issue_ (IMO) but in the release notes themselves.

Reply:
Not sure though, was this supposed to go in the issue itself?

Reply:
Yep. See https://github.com/ipfs/js-ipfs/issues/2192